mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +32,72 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Image respond to clicks
+   */
+  isZoomable?: boolean;
+
+  /**
+   * Large Image
+   */
+  isZoomed?: boolean;
```

Comment:
Looks like this props required for programmatically change image size (open in modal) from outside.  We don't need it for now I think, let's remove and keep only internal state for such puprpose.

Reply:
I removed this line because it is no longer needed